core/src/main/java/com/yahoo/ycsb/generator/ExtremeValueGenerator.java

```diff
@@ -0,0 +1,84 @@
+/**                                                                                                                                                                                
+ * Added by Min Fu.                                                                                                                                                 
```

Comment:
File needs a license header.

Git log will show who adeed the file. If yoy want an additional notice, use an author tag.